src/python/grpcio_reflection/grpc_reflection/v1alpha/reflection.py

```diff
@@ -84,13 +84,18 @@ def _file_containing_symbol(self, fully_qualified_name):
         else:
             return _file_descriptor_response(descriptor)
 
-    def _file_containing_extension(containing_type, extension_number):
-        # TODO(atash) Python protobuf currently doesn't support querying extensions.
-        # https://github.com/google/protobuf/issues/2248
-        return reflection_pb2.ServerReflectionResponse(
-            error_response=reflection_pb2.ErrorResponse(
-                error_code=grpc.StatusCode.UNIMPLEMENTED.value[0],
-                error_message=grpc.StatusCode.UNIMPLMENTED.value[1].encode(),))
+    def _file_containing_extension(self, containing_type, extension_number):
+        try:
+            pool = self._pool
```

Comment:
I preferred not repeating `self._pool` on subsequent lines, but don't feel too strongly about it. done.